.travis.sh

```diff
@@ -83,7 +83,7 @@ end script
       sudo restart docker
     fi
 
-    if [[ "$DOCKER_VERSION" =~ ^1\.12\..* ]]; then
```

Comment:
might be more maintainable long-term to add this as a param to the matrix list in travis.yml, like `ENABLE_SWARM=1`